x-pack/plugins/dashboard_enhanced/public/services/drilldowns/abstract_dashboard_drilldown/components/dashboard_drilldown_config/i18n.ts

```diff
@@ -27,3 +27,10 @@ export const txtUseCurrentDateRange = i18n.translate(
     defaultMessage: 'Use date range from origin dashboard',
   }
 );
+
+export const txtOpenInNewTab = i18n.translate(
+  'xpack.dashboard.components.DashboardDrilldownConfig.openInNewTab',
+  {
+    defaultMessage: 'Open in new tab',
```

Comment:
I think we might want to change the wording of this to be more clear on what this means - perhaps `Open drilldown in new tab by default`, or something like that. @KOTungseth Thoughts?

Reply:
Since users are opening a dashboard by _using_ the drilldown, I would use `Open dashboard in new tab`, and also leave out the `by default` bit.

Reply:
Ahh, that's perfect! Thanks so much 👍